config/database.yml

```diff
@@ -1,33 +1,23 @@
-development:
+defaults: &defaults
   adapter: postgresql
   encoding: unicode
-  database: open_food_network_dev
   pool: 5
   host: localhost
-  username: ofn
-  password: f00d
+  username: <%= ENV.fetch('DB_USERNAME', 'ofn') %>
+  password: <%= ENV.fetch('DB_PASSWORD', 'f00d') %>
+
+development:
+  <<: *defaults
+  database: open_food_network_dev
 
 test:
-  adapter: postgresql
-  encoding: unicode
+  <<: *defaults
   database: open_food_network_test
```

Comment:
If the database names were ENV vars as well, we could use a single file in both repos and remove the duplication from ofn-install, right? Would that be preferable?

https://github.com/openfoodfoundation/ofn-install/blob/master/roles/app/templates/postgresql.yml.j2

Reply:
Yes, that would be great!

By the way, the merge conflict here is with the merged docker pr which sets the database host to `db`. That fails in our current dev setup where it's `localhost`. Maybe we should take ownership of this PR while Pau is away and fix the config.

Reply:
We can make a separate issue for this in ofn-install.